dataprofiler/tests/data_readers/test_csv_data.py

```diff
@@ -530,29 +517,6 @@ def test_data_formats(self):
                 + "['dataframe', 'records']",
             )
 
-    def test_data_formats_when_sampled(self):
```

Comment:
I don't think the functionality covered in this test is affected by reservoir sampling.

Reply:
`input_data_obj = Data(input_file["path"], options={"sample_nrows": 100})`

at the very least remove try except

Reply:
i see now why the removal.